readme/BREAKING_CHANGES.markdown

```diff
@@ -94,3 +94,34 @@ This change was made to align with Adobe dropping support for Flash in December
 31, 2020 and browsers removing Flash support in upcoming versions.
 
 ---------------------------------------
+
+### The /portal/flash path is no longer available
+- **Date:** 2020-Oct-13
+- **JIRA Ticket:** [LPS-121733](https://issues.liferay.com/browse/LPS-121733)
+
+#### What changed?
+
+The public path `/portal/flash` that could be used to play an Adobe Flash movie
+passing the movie URL as a parameter has been removed.
+
+Additionally, the property and accessors have been removed from `ThemeDisplay`
+and are no longer accesible.
+
+#### Who is affected?
+
+This affects people that were using the path `/c/portal/flash` directly to show
+pages with Adobe Flash content.
+
+#### How should I update my code?
+
+A direct code update is not possible. One possible solution would be to create
+a custom page simulating to simulate the old behaviour and read the different
+movie parameters from the URL and then instantiate it using the common means
+for Adobe Flash reproduction.
+
+#### Why was this change made?
+
+This change was made to align with Adobe dropping support for Flash in December
+31, 2020 and browsers removing Flash support in upcoming versions.
```

Comment:
If you link the other one, link this one too.

Reply:
Will do, or maybe won't

Reply:
Did it too